browser/src/VariantList/Variants.tsx

```diff
@@ -211,15 +215,19 @@ const Variants = ({
     variantSearched: Variant[],
     variantsTableColumns: any
   ) {
-    const searchedVariants = getFilteredVariants(searchFilter, variantSearched, variantsTableColumns)
-  
+    const searchedVariants = getFilteredVariants(
+      searchFilter,
+      variantSearched,
+      variantsTableColumns
+    )
+
     if (searchedVariants.length > 0) {
       const firstVariant = searchedVariants[0]
-      const firstIndex = variants.findIndex((variant: Variant) => variant.pos === firstVariant.pos)
+      const firstIndex = renderedVariants.findIndex((variant: Variant) => variant.pos === firstVariant.pos)
       if (positionLastClicked !== null && firstVariant.pos < positionLastClicked) {
```

Comment:
I believe there still exists a case in which a user scrolls past the variant in the table below, without ever clicking the bubble track (I'm not entirely sure what to call that track). In that case, if the user searches for a variant that is above the currently displayed window in the table, this `if` statement will not be triggered, and the table will jump to `firstIndex + 10`, rather than `firstIndex - 10`.

I'd suggest comparing the value of `firstIndex` to the first value of `visibleVariantWindow[0]` (an array containing the first and last indices currently visible in the table and bubble track) piece of state that is defined on line `154`. Locally, when making this change, this resolved the inconsistent behavior I was observing.

Reply:
That's a great point and thanks for suggesting using that array of the visible variant window. Logically this feature is connected with the table view rather than the feature on the page about clicking the variants, so this makes more sense to use as a comparer for the edge cases.